Documentation/design.md

```diff
@@ -10,10 +10,13 @@ This document describes the design and interaction between the custom resource d
 The custom resources that the Prometheus Operator introduces are:
 
 * `Prometheus`
-* `ServiceMonitor`
-* `PodMonitor` 
 * `Alertmanager`
+* `ThanosRuler`
```

Comment:
Nit: Any specif reason for the ordering? Otherwise might make sense to have it be alphabetic?

Reply:
I used the same order as the Readme.MD. I assume they were ordered this way (I can also add those categories if it helps):

* Service Deployments (Prom, AM, Thanos)
* Monitors (ServiceMonitors, PodMonitors, Probes)
* Config (PrometheusRule, AMConfig)